common/models/order.js

```diff
@@ -59,7 +59,7 @@ module.exports = function (Order) {
           fees = calculateStripeFees(total);
         }
 
-        order.total = total + fees;
+        order.total = total + fees + order.charitableDonation;
```

Comment:
There is nothing stopping `order.charitableDonation` being negative. This needs checked!